client/js/components/month/month.js

```diff
@@ -1,15 +1,9 @@
 /** @jsx React.DOM */
 var React = require('react');
 var MonthEventsContainer = require('./month-events-container.js');
-var MonthStore = require('../../stores/month-store.js');
-var MonthHeader = require('./month-header.js');
-var MonthData = require('../../monthData.js');
-
-function monthData () {
-  return {
-    gigs: MonthStore.getMonthData()
-  };
-}
+var MonthStore = require('../../stores/month-store');
+var MonthHeader = require('./month-header');
+var MonthData = require('../../monthData');
```

Comment:
the following suggested change may merit it's own PR to get flux initialization in place: 

monthData shouldn't be required into the month component. instead, trigger a ViewAction to get the data on componentWillMount or componentDidMount after adding the listener for the store. getInitialState should return an object with the default properties for this.state (here it could just be an empty object). In each subcomponent, we should use getDefaultProps to set defaults.